projects/noloan/app/src/main/java/hasadna/noloan/mainactivity/MainActivity.java

```diff
@@ -73,26 +74,43 @@ protected void onCreate(Bundle savedInstanceState) {
 
     // Reading Sms and spams
     List<SmsMessage> messages = readSmsFromDevice();
-    List<SmsMessage> spam = SpamHolder.getInstance().getSpam();
+    List<SmsMessage> spam = DBMessagesHolder.getInstance().getSpam();
+    List<SmsMessage> suggestedSpam = DBMessagesHolder.getInstance().getSuggestions();
```

Comment:
How about `suggestions`? I'm borderline on this one.

Reply:
Yes, same here, it was `suggestions` before :) 
:+1: